app/views/claims/support/schools/claims/show.html.erb

```diff
@@ -22,6 +22,13 @@
           <% summary_list.with_row do |row| %>
             <% row.with_key(text: Claims::Claim.human_attribute_name(:accredited_provider)) %>
             <% row.with_value(text: @claim.provider.name) %>
+            <% if policy(@claim).draft? %>
+              <% row.with_action(text: t("change"),
+                                 href: edit_claims_support_school_claim_path(@school, @claim),
+                                 html_attributes: {
+                                    class: "govuk-link--no-visited-state",
+                                    }) %>
+            <% end %>
```

Comment:
This looks _strange_ to me. Possibly the policy is being misused.

`policy(_).draft?` means "Can we perform the action "draft" on this resource?". Is that how we're intending to use it here?

I'm worried we're checking if we're attempting to check if this record is a draft.

Reply:
Yeah, probably. It looks to me that this is checking whether a user can "edit" the claim, right?

I _think_ we already use the `edit` actions in the create flow too, unfortunately. Maybe we name this new one `change?` to correspond with the link's text?

cc: @CatalinVoineag